deploy/roles/r-core/meta/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+dependencies:
+  - {role: epel, when: "use_epel and not 'production' in group_names"}
+  - devtools
```

Comment:
@lel99999 `devtools` should be removed, as it is not a depdency of `r-core`, just `r-libs`